### PR TITLE
Footer bjorn

### PR DESCRIPTION
**Pull request by Bjunkz**
footer for mobile, tablet and desktop

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -82,6 +82,21 @@ <h1>Rädda Bina!</h1>
         </div>
 
         <div class="footer">   
+            <img class="some-icons" src="img/some-icons.svg" alt="facebook and instagram icons">
```
Comment (postmodernistx) [inline index.html:103]: Alt text should probably be in Swedish as the rest of the website is in Swedish? :)

Also, add `width` and `height` attributes to the [image tag](https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/).

File: index.html [hunk 2]
```diff
@@ -82,6 +82,21 @@ <h1>Rädda Bina!</h1>
...
+            <div class="footer-large">
+                <div class="news-letter-signup">
+                    <h4>Nyhetsbrev?</h4>
+                    <div class="news-letter-input-container">
+                        <input class="news-letter-input" type="text" name="mail" id="mail" placeholder="MAILADDRESS                  ">
```
Comment (postmodernistx) [inline index.html:108]: There's a lot of space after "MAILADRESS"?

Comment (postmodernistx) [reply index.html:108]: Also, should the type be "email" instead of "text" to improve usability on mobile?

File: index.html [hunk 3]
```diff
@@ -82,6 +82,21 @@ <h1>Rädda Bina!</h1>
...
+                        <button class="news-letter-signup-button"><img src="img/arrow.svg" alt="arrow icon"></button>
```
Comment (postmodernistx) [inline index.html:109]: This alt text in Swedish as well :) And width & height.

File: style.css [hunk 4]
```diff
@@ -99,6 +99,60 @@ h1 {
...
 
+.footer{
+    background-color: #F1C912;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: nowrap;
+    padding: 10vh 5vw;
+}
+.some-icons{
```
Comment (postmodernistx) [inline style.css:110]: Some inconsistency between having a space before the curly brace, and not having one. Not a biggie, but good for consistency :)

File: style.css [hunk 5]
```diff
@@ -99,6 +99,60 @@ h1 {
...
+    padding: 1vh;
+}
+
+.news-letter-input-container {
+    position: relative;
+    left: 3vw;
+}
+.footer ::placeholder {
+    color: #000000;
+    font-size: 1.3rem;
```
Comment (postmodernistx) [inline style.css:133]: Good with the font size in rem!

**Review by postmodernistx — APPROVED** [review]
Looks great! 👍

**Comment by cutlasskiwi** [comment]
Ser bra ut Björn. Snyggt jobbat!

**Pull request merged**